src/core/Ros.js

```diff
@@ -125,6 +125,29 @@ Ros.prototype.getTopics = function(callback) {
 };
 
 /**
+ * Retrieves Topics in ROS as an array as specific type
+ *
+ * @param topic_type topic type to find:
+ * @param callback function with params:
+ *   * topics - Array of topic names
+ */
+Ros.prototype.getTopicsForType = function(topic_type, callback) {
```

Comment:
Can you camel case variables (`topic_type` and `service_type` below) for consistency with the JS libraries? Thanks!